apps/docs/src/examples/inputs/TagsSuggestions.tsx

```diff
@@ -75,38 +75,45 @@ export default function Demo() {
           />
         }
         classes={{
-          tagsList: "h-32px pr-0 overflow-hidden",
+          tagsList: "h-32px pr-0! overflow-hidden",
```

Comment:
this seems to do nothing
we should avoid using `!important` unless it's really necessary